src/containers/BooksForm.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { createBook } from '../actions';
+
+const styles = {
```

Comment:
Let us be consistent in the whole project with how we include our styles. In other components we have been using css modules. Then here we are using it "in-file". I strongly suggest we create a `BooksForm.css` file to contain these styles for this component.

Reply:
I wanna remind you guys about this :) Let's have a file that will house the styles for BooksForm